pyproject.toml

```diff
@@ -70,7 +69,8 @@ coverage = ">=7.2.3"
 # coveralls = ">=3.3.1"
 flake8 = ">=3.9.2"
 flaky = ">=3.7.0"
-pip-licenses = "^3.5.5"
+pip-licenses = "^4.3.3"
+pip = "23.2.1"
```

Comment:
I would order this before `pip-licenses`, but that's minor.

Is it really a good idea to list `pip` here? Is `poetry` undoing the version we load in the `Makefile`? (In that case, maybe it is necessary to list it here. But it seems like something we may regret having to maintain.

Further, do you really mean to pin it to so specific a version? If so, could you add a comment saying why so we can know when to unpin it?

Reply:
You're right, I had that (pip) in there temporarily - I will remove it. Thanks.